jbpm-designer-utilities/src/main/java/org/jbpm/designer/utilities/svginline/RunSvgInline.java

```diff
@@ -18,7 +18,7 @@
 
 public class RunSvgInline {
 
-    public static final String SS_IN_FILE = "/home/jeremy/Documents/gitRepositories/droolsjbpm/jbpm-designer/jbpm-designer-client/src/main/resources/org/jbpm/designer/public/stencilsets/bpmn2.0jbpm/stencildata/bpmn2.0jbpm.orig";
+    public static final String SS_IN_FILE = "/home/jeremy/Documents/gitRepositories/kiegroup/jbpm-designer/jbpm-designer-client/src/main/resources/org/jbpm/designer/public/stencilsets/bpmn2.0jbpm/stencildata/bpmn2.0jbpm.orig";
     public static final String SS_OUT_FILE = "/home/jeremy/Temp/bpmn2.0jbpm.orig.svginline";
```

Comment:
Hi @hasys,
For a note about how they're used, see https://github.com/kiegroup/jbpm-designer/blob/master/jbpm-designer-utilities/README.md

I guess this should really be cleaned up, but as this module is very rarely used, it would be low priority.